config/services.go

```diff
@@ -323,6 +323,22 @@ func GetVirtualSpaces() []string {
 	return getVspaceKeys(viper.GetStringMap("virtual_spaces"))
 }
 
+func GetVirtualSpace(spaceName string) (*base.VirtualSpace, error) {
+
```

Comment:
You'll probably get a linter error for this empty line.

Reply:
exact. Fixed